### ISSUE TITLE
About portfolio

### ISSUE THREAD

**Issue by Ali82APS** [codeschool-projects/HTMLPortfolioProject#423]
portfolio helps companies to understand the programmer better.

**Comment by 118abhi** [comment]
Yes if you build a more dynamic portfolio thann  it  yoour chance is very high

**Comment by garima-203** [comment]
Portfolio is a kind of way to showcase your Capability , Creativity, Experience , Knowledge as a devops. It is the best way to showcase your past experience (projects)  and to know more about you to others

**Comment by Banbastic** [comment]
I've seen great design inspirations in this portfolio
